<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-CH4OCH9Q" failures="2" tests="8" name="FirefoxTest" time="340.715" errors="0" timestamp="2025-09-16T09:36:42 IST">
  <testcase classname="testscript.LoginTest" name="verifyLoginWithValidCredentials" time="0.052">
    <skipped/>
  </testcase> <!-- verifyLoginWithValidCredentials -->
  <testcase classname="testscript.LoginTest" name="VerifyLoginWithInvalidUsername" time="0.045">
    <failure type="java.io.IOException" message="Truncated ZIP file">
      <![CDATA[java.io.IOException: Truncated ZIP file
at org.apache.commons.compress.archivers.zip.ZipArchiveInputStream.readDeflated(ZipArchiveInputStream.java:1109)
at org.apache.commons.compress.archivers.zip.ZipArchiveInputStream.read(ZipArchiveInputStream.java:1028)
at java.base/java.io.FilterInputStream.read(FilterInputStream.java:119)
at org.apache.poi.openxml4j.util.ZipArchiveThresholdInputStream.read(ZipArchiveThresholdInputStream.java:78)
at org.apache.poi.util.IOUtils.toByteArray(IOUtils.java:245)
at org.apache.poi.util.IOUtils.toByteArray(IOUtils.java:208)
at org.apache.poi.openxml4j.util.ZipArchiveFakeEntry.<init>(ZipArchiveFakeEntry.java:83)
at org.apache.poi.openxml4j.util.ZipInputStreamZipEntrySource.<init>(ZipInputStreamZipEntrySource.java:98)
at org.apache.poi.openxml4j.opc.ZipPackage.<init>(ZipPackage.java:157)
at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:351)
at org.apache.poi.ooxml.util.PackageHelper.open(PackageHelper.java:64)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:313)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:287)
at utilities.ExcelUtility.getStringData(ExcelUtility.java:28)
at testscript.LoginTest.VerifyLoginWithInvalidUsername(LoginTest.java:55)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- VerifyLoginWithInvalidUsername -->
  <testcase classname="testscript.LoginTest" name="VerifyLoginWithInvalidPassword" time="0.0">
    <skipped/>
  </testcase> <!-- VerifyLoginWithInvalidPassword -->
  <testcase classname="testscript.LoginTest" name="verifyLoginWithValidCredentials" time="5.191">
    <skipped/>
  </testcase> <!-- verifyLoginWithValidCredentials -->
  <testcase classname="testscript.LoginTest" name="VerifyLoginWithInvalidUsernameandPassword" time="0.0">
    <skipped/>
  </testcase> <!-- VerifyLoginWithInvalidUsernameandPassword -->
  <testcase classname="testscript.LoginTest" name="verifyLoginWithValidCredentials" time="0.251">
    <failure type="org.openqa.selenium.remote.UnreachableBrowserException" message="Error communicating with the remote browser. It may have died.
Build info: version: &amp;apos;4.16.1&amp;apos;, revision: &amp;apos;9b4c83354e&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;21.0.7&amp;apos;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Command: [7e5fa244-6340-4b50-8e15-b4317ba27d33, findElement [using, value]]
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 142.0.1, moz:accessibilityChecks: false, moz:buildID: 20250827004350, moz:geckodriverVersion: 0.36.0, moz:headless: false, moz:platformVersion: 10.0, moz:processID: 6136, moz:profile: C:\Users\sheri\AppData\Loca..., moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, userAgent: Mozilla/5.0 (Windows NT 10....}
Session ID: 7e5fa244-6340-4b50-8e15-b4317ba27d33">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: '4.16.1', revision: '9b4c83354e'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.7'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Command: [7e5fa244-6340-4b50-8e15-b4317ba27d33, findElement [using, value]]
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 142.0.1, moz:accessibilityChecks: false, moz:buildID: 20250827004350, moz:geckodriverVersion: 0.36.0, moz:headless: false, moz:platformVersion: 10.0, moz:processID: 6136, moz:profile: C:\Users\sheri\AppData\Loca..., moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, userAgent: Mozilla/5.0 (Windows NT 10....}
Session ID: 7e5fa244-6340-4b50-8e15-b4317ba27d33
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:550)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:360)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:354)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at jdk.proxy2/jdk.proxy2.$Proxy16.sendKeys(Unknown Source)
at utilities.PageUtility.sendDataToElement(PageUtility.java:12)
at pages.LoginPage.enterUsername(LoginPage.java:26)
at testscript.LoginTest.verifyLoginWithValidCredentials(LoginTest.java:27)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.io.UncheckedIOException: java.net.ConnectException
at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:442)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute(JdkHttpClient.java:358)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:188)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:200)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:175)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:523)
... 26 more
Caused by: java.net.ConnectException
at java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1041)
at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:227)
at java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:280)
at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$2(PlainHttpConnection.java:238)
at java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)
at java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)
at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)
... 3 more
Caused by: java.nio.channels.ClosedChannelException
at java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:202)
at java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:786)
at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:874)
at java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:210)
at java.base/java.security.AccessController.doPrivileged(AccessController.java:571)
at java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:212)
... 9 more
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- verifyLoginWithValidCredentials -->
  <testcase classname="testscript.LoginTest" name="VerifyLoginWithInvalidUsernameandPassword" time="0.0">
    <skipped/>
  </testcase> <!-- VerifyLoginWithInvalidUsernameandPassword -->
  <testcase classname="testscript.LoginTest" name="VerifyLoginWithInvalidUsernameandPassword" time="0.0">
    <skipped/>
  </testcase> <!-- VerifyLoginWithInvalidUsernameandPassword -->
</testsuite> <!-- FirefoxTest -->
